pymod/test/__init__.py

```diff
@@ -1,17 +1,7 @@
 # noqa
+from .test_chemistry import *
+from .test_geometry import *
+import geometry
+import chemistry
 import sys
 sys.path.insert(0, '..')
-import chemistry
-import geometry
-
-from .test_geometry import (
-        TestCoordinate,
-        TestSurface,
-        TestSurfacePoint,
-        )
-
-from .test_chemistry import (
-        TestAtom,
-        TestAtomList,
-        TestCellList,
-        )
```

Comment:
Well this is only the `__init__.py` for the testing, so we do want to bring in all submodules. If we don't want to run them, we just annotate them with `@pytest.mark.skip(reason='reason')` at the test level. That's why I `import *`'ed the tests here. I'd hate to miss a test because we forgot to import it.

Reply:
As per Nathan's comment, considering resolved. Feel free to unresolve if this still doesn't sit well with you!